frontend/src/components/videoControls/components/Fullscreen/index.js

```diff
@@ -4,23 +4,39 @@ import { Tooltip, SvgIcon, IconButton } from '@mui/material'
 import { useTheme } from '@mui/material/styles'
 import {ReactComponent as FullscreenIcon} from './fullscreen.svg'
 import {ReactComponent as FullscreenExitIcon} from './fullscreenexit.svg'
-import { useStore } from '../../../store'
+import { useStore } from '../../../../store'
 
 const Fullscreen = observer(() => {
-  const {videoPlayerStore: { isFullscreen, toggleFullscreen }} = useStore()
+  const {
+    videoPlayerStore: { isFullscreen, toggleFullscreen},
+    rosStore: { isWSConnected, isTeleopReady },
+  } = useStore()
 
   const fullscreenIcon = () => {
     return isFullscreen ? FullscreenExitIcon : FullscreenIcon
   }
 
   const theme = useTheme()
 
+  const connected = isWSConnected && isTeleopReady
```

Comment:
It seems like these two constants are duplicated in several components. Try to optimize it.

Reply:
optimized for components related to CameraControls